README.md

```diff
@@ -63,12 +63,17 @@ axe.run(function (err, results) {
 ```
 ## Supported Browsers
 
-The [aXe API](doc/API.md) supports the following browsers:
+The [aXe API](doc/API.md) fully supports the following browsers:
 
-* Internet Explorer v9, 10, 11
+* Microsoft Edge v40 and above
 * Google Chrome v42 and above
 * Mozilla Firefox v38 and above
 * Apple Safari v7 and above
+* Internet Explorer v9, 10, 11
```

Comment:
I'd be massively in favour of dropping support for all but IE 11.

Reply:
It is still very widely used in government

Reply:
"very widely" seems a bit much. We have analytics on DQU right? How much IE9 + IE10 users do we get on the axe-core pages there?

Reply:
I looked up our analytics. We've got 0.3% traffic on IE9, we have nothing on IE10. As far as older IE support, there is still 1.9% on IE8, and 0.4% on IE7, but we've never supported those. I think it's pretty reasonable to drop IE9 & 10 at this point.

Reply:
The discussion/decision about whether to drop support for IE<11 should not hold up this PR